lib/foreman_openscap/helper.rb

```diff
@@ -16,25 +16,15 @@ def self.get_asset(cname, policy_id)
   end
 
   def self.find_name_or_uuid_by_host(host)
-    if defined?(::Katello::System)
-      host.content_host.uuid
-    else
-      host.name
-    end
+    host.name
```

Comment:
what about existing reports that use the cert uuid as their key? i think the test should check if there is a content facet and then use uuid or name?
also, its critical to find if this is used in migrations, as the content facet migration might happen after making this condition not relevant in migration.